src/app/routes/upload.py

```diff
@@ -0,0 +1,25 @@
+from flask import render_template, request, flash
+from werkzeug.utils import secure_filename
+
+from app import routes_logger
+from app import upload_service
+
+
+def route():
+    results = []
+    if request.method == 'POST':
+        enable_encryption = 'enableEncryption' in request.form

```

Comment:
Can you double check that this works as expected? I believe this is how I originally had it but it's always there and not just when it's selected. That might have been because of how I handled the toggle buttons though.

Reply:
Did you double check that this is correctly True only when the box is checked?

Reply:
double checked, the value is false when not selected and true when selected. It also stays selected when selected.